react-native/react/engine/windows-hack.js

```diff
@@ -0,0 +1,16 @@
+import net from 'net'
+
+export default function windowsHack () {
+  // This net.connect() is a heinous hack.
+  //
+  // On Windows, but *only* in the renderer thread, our RPC connection
```

Comment:
Not sure how to do that, will spend a few mins looking.  (It's harmless to the main thread, of course.)